migrations/Version202307251344000003_taoAdvancedSearch.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoAdvancedSearch\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\taoAdvancedSearch\model\SearchEngine\Contract\IndexerInterface;
+use oat\taoAdvancedSearch\scripts\tools\IndexMigration;
+
+final class Version202307251344000003_taoAdvancedSearch extends AbstractMigration
+{
+    private const INDEX_UPDATE_BODY = '{"properties": {"test_qti_structure": {"type": "object", "enabled": false}}}';
```

Comment:
Note for reviewers: This ends up doing a PUT request to add the new properties, see https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-put-mapping.html & https://github.com/oat-sa/extension-tao-advanced-search/blob/28fd95154126a891d30fe16e6d9ef024f7e14ffb/scripts/tools/IndexMigration.php :ok_hand: